src/web/app/pages-student/student-course-details-page/student-course-details-page.component.html

```diff
@@ -152,7 +152,7 @@ <h4 class="font-weight-bold">Teammates</h4>
             <tr>
               <td class="font-weight-bold">Gender</td>
               <td>
-                <span *ngIf="teammateProfile.gender != 'other'">{{teammateProfile.gender}}</span>
+                <span *ngIf="teammateProfile.gender != Gender.OTHER">{{teammateProfile.gender}}</span>
```

Comment:
You forget to export Gender like what you did in `student-profile-page.component.ts`?

Please check other places also.

Reply:
I've made the change. Is [this](https://github.com/TEAMMATES/teammates/pull/9440/commits/407806c0bfd9e76e85881b98de5643d16cb88f73) what you meant?

Reply:
Also, page outputs have been uploaded in the PR description.